style_editor.html

```diff
@@ -6747,7 +6747,7 @@
   constructor(ARGS) {
     super("Select transitions", ARGS);
     this.TRANSITIONS = Array.from(ARGS).slice(1);
-    this.add_arg("F", "FUNCTION", "Transition to select");
+    this.add_arg("F", "FUNCTION", "FUNCTION to select Transition");
```

Comment:
Not sure this is better.
The type isn't normally part of these comments.
Maybe "Selects which transition to use" ?